app/lib/getFileIcon/index.js

```diff
@@ -0,0 +1,2 @@
+const getFileIcon = process.platform === 'darwin' ? require('./mac') : null
```

Comment:
It would be better to export empty function, instead of `null` to avoid ... is not a function errors